data/org.servo.Servo.desktop

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
```

Comment:
This should also move to `support/flatpak`.

Reply:
These are generic Linux files with descriptions not tied to the Flatpak ecosystem (regular distributions use them). I would understand that they're not particularly useful for Servo at this moment, but in the future, they'll be needed independently from Flatpak.

I'll have a bit delayed reaction; sorry about that. Thank you for your review.